openstack/nova/templates/seed.yaml

```diff
@@ -69,7 +70,7 @@ spec:
     - interface: public
       region: '{{ .Values.global.region }}'
       url: 'https://{{include "placement_api_endpoint_host_public" .}}:{{.Values.global.placementApiPortPublic}}'
-
+{{ end }}
   domains:
   - name: Default
     users:
```

Comment:
The role-assignments down below don't have a `if .Values.placement.enabled`. Is that by choice? If yes, please add the reasoning to the commit-message.